### PR TITLE
rio: 0.0.7 -> 0.0.8

### PR DESCRIPTION
**Pull request by otavio**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/terminal-emulators/rio/default.nix [hunk 1]
```diff
@@ -18,7 +18,7 @@
 , libXrandr
```
Comment (oluceps) [inline pkgs/applications/terminal-emulators/rio/default.nix:18]: Core dumped on sway, seems to be missing `libxkbcommon`.
```
> RUST_BACKTRACE=full ./rio
[INFO] rio [setup_environment_variables] terminfo: xterm-256color
thread 'main' panicked at 'internal error: entered unreachable code', /build/rio-0.0.8-vendor.tar.gz/winit/src/platform_impl/linux/wayland/window/mod.rs:333:77
stack backtrace:
   0:     0x582f5648feae - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::had616a77c932ab1b
   1:     0x582f562c232f - core::fmt::write::h3a35c0754863b3d5
   2:     0x582f5649c5e5 - std::io::Write::write_fmt::h502c0f36b9ea0da9
   3:     0x582f5648fc75 - std::sys_common::backtrace::print::h90cd8adaf174c598
   4:     0x582f5649578e - std::panicking::default_hook::{{closure}}::ha88f96a3c80996f9
   5:     0x582f56495450 - std::panicking::default_hook::haf5a79a7bf67f396
   6:     0x582f56495cab - std::panicking::rust_panic_with_hook::h69ad028564cd3ed4
   7:     0x582f56490185 - std::panicking::begin_panic_handler::{{closure}}::h8ea911da5e199679
   8:     0x582f5648ffb6 - std::sys_common::backtrace::__rust_end_short_backtrace::h02258dadc83c0b77
   9:     0x582f564959b2 - rust_begin_unwind
  10:     0x582f5622c113 - core::panicking::panic_fmt::h55a49ce1199dc412
  11:     0x582f5622c1ad - core::panicking::panic::h999ed26cf2da3d31
  12:     0x582f5640577e - winit::platform_impl::platform::wayland::window::Window::new::{{closure}}::h62b0f366d969db08
  13:     0x582f564f92ce - scoped_tls::ScopedKey<T>::with::h09c0db5c8a3a2972
  14:     0x582f564fb777 - std::panicking::try::h7377560dded418a7
  15:     0x582f56506f76 - wayland_client::imp::proxy::proxy_dispatcher::h91361fcbe80081ae
  16:     0x7a6afc4f698a - wl_closure_dispatch
  17:     0x7a6afc4f2b71 - dispatch_event.isra.0
  18:     0x7a6afc4f4524 - wl_display_dispatch_queue_pending
  19:     0x7a6afc4f4adf - wl_display_roundtrip_queue
  20:     0x582f56401532 - winit::platform_impl::platform::Window::new::hc0797a1be5ceeaf6
  21:     0x582f5642418b - rio::sequencer::Sequencer::run::{{closure}}::hf27a24e56570dc5c
  22:     0x582f564303da - rio::main::{{closure}}::h455700016fd43ba0
  23:     0x582f5642dda1 - rio::main::h09217f68448e31f4
  24:     0x582f563dc983 - std::sys_common::backtrace::__rust_begin_short_backtrace::hcbe89ed565569001
  25:     0x582f563dd137 - std::rt::lang_start::{{closure}}::h54c245dba2e4e1f9
  26:     0x582f564958d2 - std::panicking::try::h2fded24329a9b820
  27:     0x582f5647ebdb - std::rt::lang_start_internal::hbbe6eb94b9b031b1
  28:     0x582f56430b5c - main
  29:     0x7a6aff657ace - __libc_start_call_main
  30:     0x7a6aff657b89 - __libc_start_main@@GLIBC_2.34
  31:     0x582f5624ff85 - _start
  32:                0x0 - <unknown>
[wayland-client error] A handler for wl_keyboard panicked.
Aborted (core dumped)
```

Comment (otavio) [reply pkgs/applications/terminal-emulators/rio/default.nix:19]: I fixed this, please check if it works for you now.

Comment (oluceps) [reply pkgs/applications/terminal-emulators/rio/default.nix:19]: It works fine on my machine now.

**Comment by oluceps** [comment]
Result of `nixpkgs-review pr 241319` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>rio</li>
  </ul>
</details>

**Review by oluceps — APPROVED** [review]

**Pull request merged**